src/folder/ui-add-folder-button/AddFolderButton.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "./AddFolderButton.module.scss";
+import classNames from "classnames/bind";
+import { ReactComponent as AddIcon } from "./add.svg";
+
+interface AddFolderButtonProps {
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+const cx = classNames.bind(styles);
+
+export const AddFolderButton: React.FC<AddFolderButtonProps> = ({ onClick }) => {
```

Comment:
💬  how about
```suggestion
export const AddFolderButton = ({ onClick }: AddFolderButtonProps) => {
```

- `React.FC`나 `FunctionComponent` 타입을 명시하지 않아도, TypeScript는 제공된 프로퍼티 타입을 통해 컴포넌트의 타입을 추론할 수 있습니다. 이 방식은 타입스크립트를 사용하는 리액트 컴포넌트에서 널리 사용되는 패턴이에요!

- `React.FC`를 사용하면 `children` 프로퍼티가 자동으로 컴포넌트의 프로퍼티 타입에 포함됩니다. 모든 컴포넌트가 children을 받지 않는 경우, 이는 불필요한 타입 확장을 의미할 수 있습니다

Reply:
React.FC로 함수 컴포넌트 타입 지정이 많이 되어있군요~! 
이 부분들 모두 체크해주시면 감사하겠습니다 👍🏻